src/upload/system/library/transbank/SDK/TransbankSdkWebpay.php

```diff
@@ -40,9 +41,9 @@ public function initTransaction($amount, $sessionId, $buyOrder, $returnUrl)
                     "token_ws" => $response->token
                 ];
             } else {
-                throw new \Exception('No se ha creado la transacción para, amount: ' . $amount . ', sessionId: ' . $sessionId . ', buyOrder: ' . $buyOrder);
+                throw new TransactionCreationException();
             }
-        } catch (\Exception $e) {
+        } catch (TransactionCreationException $e) {
```

Comment:
if you catch just this exception released in the previous line, then this catch is unnecessary and can set result instead thow `TransactionCreationException`.
For this catch, look for Transaction->create method and the possible exceptions that it may throw